### ISSUE TITLE
E-mail complete scoresheet to teams

### ISSUE THREAD

**Issue by Jheronymus** [FirstLegoLeague/fllscoring#268]
Combine the PDF generated from the XML with the actual scoring data and signature and send final result to the team.

There should be an option in the settings/administrator interface when to do so.

**Comment by kmeesters** [comment]
I think this should ideally be handled by a different module. For example a 'team-app' or 'team management' module. My motivation for this would be:
- Reduced complexity in the scoring app (prevent building a monolith)
- Less risk of breaking or blocking something in scorign
- Centralized communication handling (1 place to store email adressen, send out notification etc)
- Can also be used by other modules (judging forms for example)

I think this is perfect case for MHub to signal a new score (after verification) and send a message. Could be picked up by this new module, bundled with team info and send out.

**Comment by Jheronymus** [comment]
I agree. Setting up an outgoing e-mail address also requires additional steps that should not be part of a match/scoring setup but is much more related to a  tournament setup
